src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -61,7 +65,7 @@ public function getLocator()
      *
      * @param mixed       $resource       A Resource
      * @param string|null $type           The resource type or null if unknown
-     * @param bool        $ignoreErrors   Whether to ignore import errors or not
+     * @param int         $ignoreErrors   Whether to ignore import errors or not
```

Comment:
The description of this arguments should be updated as it's not a boolean type anymore.

Something like `How to handle import errors`

Reply:
Fixed, thanks @sstok